src/screenbot.js

```diff
@@ -49,22 +91,31 @@ var Screenbot = (function Screenbot() {
         _this.command = function (command, cb) {
           console.log("Source: " + _private.source);
 
-          _request(_endpoint(command), function(success, response) {
-
-            if (!success) return cb(false, response.error);
+          _request(_endpoint(command), function(err, result) {
+            if(err instanceof Error) return cb(err);
```

Comment:
This works fine but I do wonder if maybe you want all errors passed to the callback to be instances of `ScreenbotError`. If so, this line could be eliminated.